### ISSUE TITLE
format setting link doesn't work

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#196130]
1. search for audio cue format setting
2. click on the linked setting
3. settings editor is searching and doesn't go to it

https://github.com/microsoft/vscode/assets/29464607/eda1d3d3-fa11-4e9a-a25f-e40302f4385c
